cachi2/core/utils.py

```diff
@@ -6,12 +6,15 @@
 from pathlib import Path
 from typing import Iterator, Union
 
-from cachi2._compat.errors import UnknownHashAlgorithm
 from cachi2.core.config import get_worker_config
 
 log = logging.getLogger(__name__)
 
 
+class UnknownHashAlgorithm(ValueError):
+    """The hash algorithm is unknown by Cachi2."""
```

Comment:
This is just meant to be a "more specific ValueError", not a user-facing error type. OK to keep it in this file?

Reply:
I would probably place it to `core/errors.py` to others but I am fine keeping it here. :)

Reply:
Solved this differently. The `utils.hash_file` saves 3 lines at best, removed it instead

Reply:
(And that removes the need for an UnknownHashAlgorithm error)